core/src/test/java/model/HotelTest.java

```diff
@@ -8,41 +8,40 @@
 import java.time.LocalTime;
 import java.util.Optional;
 
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
 
 import pl.agh.edu.enums.RoomRank;
 import pl.agh.edu.json.data_extractor.JSONFilePath;
 import pl.agh.edu.model.Hotel;
 import pl.agh.edu.model.Room;
-import pl.agh.edu.model.RoomState;
 import pl.agh.edu.model.client.ClientGroup;
 
 public class HotelTest {
 
-	@Mock
+	private final ClientGroup group = mock(ClientGroup.class);
```

Comment:
wczoraj przegadaliśmy że zostawiamy same adnotacje bez instancjonowania,  ale spoko poźniej zrobię refactor wszystkiego to podmienię